CMakeLists.txt

```diff
@@ -384,6 +384,9 @@ OCV_OPTION(WITH_OPENCLAMDFFT "Include AMD OpenCL FFT library support" ON
 OCV_OPTION(WITH_OPENCLAMDBLAS "Include AMD OpenCL BLAS library support" ON
   VISIBLE_IF NOT ANDROID AND NOT IOS AND NOT WINRT
   VERIFY HAVE_CLAMDBLAS)
+OCV_OPTION(WITH_OPENCL_D3D11_NV "Include NVIDIA OpenCL D3D11 support" OFF
```

Comment:
Lets try to enable this in default builds (to avoid breaking compilation of this code in the future):
`OFF` => `(WITH_DIRECTX)`

and move this option after `WITH_DIRECTX` declaration.